membership-attribute-service/app/monitoring/CloudWatch.scala

```diff
@@ -73,10 +43,17 @@ class CreateMetrics(stage: Stage) {
     .build()
 
   def forService(service: Class[_]): Metrics = Metrics(service.getSimpleName, stage.value, cloudwatch)
+
   def expensiveForService(service: Class[_])(implicit system: ActorSystem, ec: ExecutionContext): ExpensiveMetrics =
     new ExpensiveMetrics(service.getSimpleName, stage.value, cloudwatch)
 }
 
-object CloudWatchHealth {
-  var hasPushedMetricSuccessfully = false
+class LoggingAsyncHandler extends AsyncHandler[PutMetricDataRequest, PutMetricDataResult] with StrictLogging {
+  def onError(exception: Exception) {
+    logger.info(s"CloudWatch PutMetricDataRequest error: ${exception.getMessage}}")
+  }
+
+  def onSuccess(request: PutMetricDataRequest, result: PutMetricDataResult) {
+    logger.debug("CloudWatch PutMetricDataRequest - success")
```

Comment:
Worth keeping an eye on how much this adds to the logs, it could get quite noisy

Reply:
Changed it to trace.